Mergin/validation.py

```diff
@@ -245,21 +247,23 @@ def check_attachment_widget(self):
                 if ws and ws.type() == "ExternalResource":
                     cfg = ws.config()
                     # check for relative paths
-                    if "RelativeStorage" in cfg and cfg["RelativeStorage"] == 0:
-                        self.issues.append(SingleLayerWarning(lid, Warning.ATTACHMENT_ABSOLUTE_PATH))
-                    if "DefaultRoot" in cfg:
-                        # default root should not be set to the local path
-                        if os.path.isabs(cfg["DefaultRoot"]):
-                            self.issues.append(SingleLayerWarning(lid, Warning.ATTACHMENT_LOCAL_PATH))
-
-                        # expression-based path should be set with the data-defined overrride
-                        expr = QgsExpression(cfg["DefaultRoot"])
-                        if expr.isValid():
-                            self.issues.append(SingleLayerWarning(lid, Warning.ATTACHMENT_EXPRESSION_PATH))
-
-                        # using hyperlinks for document path is not allowed when
-                        if "UseLink" in cfg:
-                            self.issues.append(SingleLayerWarning(lid, Warning.ATTACHMENT_HYPERLINK))
+                    if "RelativeStorage" in cfg:
+                        if cfg["RelativeStorage"] == 0:
+                            self.issues.append(SingleLayerWarning(lid, Warning.ATTACHMENT_ABSOLUTE_PATH))
```

Comment:
This one is questionable (not in this PR) - the app now accepts absolute paths in the form of network URL addresses. We should update the text of the validation.

Other conditions look good from the quick look, but must be properly tested (@RastoHu)